internal/pkg/addons/addons.go

```diff
@@ -0,0 +1,79 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package addons contains the service to manage addons.
+package addons
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/workspace"
+)
+
+const (
+	paramsFileName  = "params"
+	outputsFileName = "outputs"
+	indent          = "  " // two spaces
+)
+
+type workspaceService interface {
+	ReadAddonsFile(appName, fileName string) ([]byte, error)
```

Comment:
[nit] Can we call these ReadAddonFIle / ListAddonFiles? (get rid of the `s` after Addon)

Reply:
lol just changed from `ListAddonFiles` https://github.com/aws/amazon-ecs-cli-v2/pull/662#discussion_r380334188

Reply:
Oh, oky doky! I don't mind either way.

Reply:
😎 sorry David!